src/prommis/solvent_extraction/solvent_extraction.py

```diff
@@ -86,8 +86,53 @@
     useDefault,
 )
 from idaes.core.util.config import is_physical_parameter_block
+from idaes.core.initialization import ModularInitializerBase
 from idaes.models.unit_models.mscontactor import MSContactor
 
+
+class SolventExtractionInitializer(ModularInitializerBase):
+    """
+    This is a general purpose Initializer  for the Solvent Extraction unit model.
+
+    This routine calls the initializer for the internal MSContactor model.
+
+    """
+
+    CONFIG = ModularInitializerBase.CONFIG()
+
+    def initialize_main_model(
+        self,
+        model: Block,
+    ):
+        """
+        Initialization routine for MSContactor Blocks.
+
+        Args:
+            model: model to be initialized
+
+        Returns:
+            None
+        """
+        """
+        This model adds an additional constraint of the material transfer term.
+        This constraint is present outside the main mscontactor model, so this term is 
+        fixed to give a square model to the MScontactor Initializer.
```

Comment:
mscontactor should be spelled like MSContactor when referring to it in the doc strings (for consistency)